saleor/plugins/webhook/tasks.py

```diff
@@ -294,9 +296,11 @@ def send_webhook_request_async(self, event_delivery_id):
             delivery.id,
         )
     except ValueError as e:
-        response = WebhookResponse(content=str(e))
+        response = WebhookResponse(content=str(e), status=EventDeliveryStatus.FAILED)
         attempt_update(attempt, response)
         delivery_update(delivery=delivery, status=EventDeliveryStatus.FAILED)
+    if delivery.status == EventDeliveryStatus.SUCCESS:
+        clear_successful_delivery(delivery)
```

Comment:
This if statement is redundant because we have the same in `clear_successful_delivery`